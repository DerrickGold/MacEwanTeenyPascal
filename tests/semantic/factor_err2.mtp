(*try assinging a string literal*)
var a : integer;
begin
	a := '1'
end.