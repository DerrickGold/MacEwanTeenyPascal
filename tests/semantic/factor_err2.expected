
Semantic Error [line 4]: value: "literal0"
	Unexpected r-value type: 
Expected type(s) 'Integer'; found 'String' instead.
